changes/unreleased/README.md

```diff
@@ -0,0 +1,18 @@
+# Changelogs
+
+To add a changelog, add a Markdown file to a subdirectory depending on the effort required to update to
+that version:
+
+- [Major](./major): A large effort. This will cause a version bump from e.g. 0.1.2 to 1.0.0
+- [Medium](./medium): Some effort. This will cause a version bump from e.g. 0.1.2 to 1.2.0
+- [Minor](./minor): Little/no effort. This will cause a version bump from e.g. 0.1.2 to 0.1.3
+
+Therefore, the versions use [EffVer](https://jacobtomlinson.dev/effver/).
```

Comment:
This basically isn't semver. I'm wondering if we expose an api (https://github.com/NixOS/nixpkgs-check-by-name/issues/5) would this be reconsidered?

Reply:
We do already expose an API, but it's very basic: https://github.com/NixOS/nixpkgs-check-by-name/blob/62d820f35135edd7e04ede728d35a7d3bef61d83/src/main.rs#L23-L35

After seeing [EffVer](https://jacobtomlinson.dev/effver/) being discussed [on lobste.rs](https://lobste.rs/s/ynhu2s/effver_version_your_code_by_effort), I'm kind of sold on it because it makes a lot of sense and want to use it in practice.

Reply:
>discussed [on lobste.rs](https://lobste.rs/s/ynhu2s/effver_version_your_code_by_effort)

Are you a user there? Could I get an invite? :)

Reply:
Sure, I'll PM you :D